### DISCUSSION TITLE
Using useControl and deck.gl

### DISCUSSION THREAD

**Discussion by zrichardet** [uber/react-map-gl#1862] [Q&A]
I have an application that is using deck.gl v8.7.8 and react-map-gl v7.0.11. I am using the new useControl hook to use the MapboxDraw library directly that is replacing the react-map-gl-draw library that is no longer supported with the newest version of react-map-gl. 

I lose the ability to draw polygons on the map when using deck.gl to render my layers on the map because the deck.gl overlay renders on top of the react-map-gl map. I was looking for any work around for this issue so that I am able to use both deck.gl and the native MapboxDraw functions together?

**Comment by Pessimistress** [comment]
`@deck.gl/mapbox@8.8.0-alpha.2` has a new export `MapboxOverlay` which implements Mapbox's IControl.

https://github.com/visgl/deck.gl/blob/master/docs/api-reference/mapbox/mapbox-overlay.md
